src/precompile.jl

```diff
@@ -0,0 +1,38 @@
+function _precompile_()
+    ccall(:jl_generating_output, Cint, ()) == 1 || return nothing
+    # Memento
+    isdefined(Memento, Symbol("##config!#72")) && precompile(Tuple{getfield(Memento, Symbol("##config!#72")), Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}}, typeof(Memento.config!), String, String})
```

Comment:
Is there a way to get this using `Core.kwfunc`?

Reply:
When was that added?

Reply:
Alright, using `kwfunc` seems to work on both 1.5 and the lts version of julia, so I've changed that.